jetbrains-core/src/software/aws/toolkits/jetbrains/core/executables/ExecutableSettings.kt

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.executables
+
+import com.intellij.openapi.components.BaseState
+import com.intellij.openapi.components.RoamingType
+import com.intellij.openapi.components.SimplePersistentStateComponent
+import com.intellij.openapi.components.State
+import com.intellij.openapi.components.Storage
+import com.intellij.openapi.components.service
+import com.intellij.util.xmlb.annotations.Attribute
+import com.intellij.util.xmlb.annotations.Property
+import com.intellij.util.xmlb.annotations.Tag
+
+@State(name = "executables2", storages = [Storage("aws.xml", roamingType = RoamingType.DISABLED)])
```

Comment:
I turned off Roaming, there is no reason we should sync executable paths across computers IMO